routes/admin.js

```diff
@@ -10,67 +10,28 @@ router.get('/owners', async (req, res, next) => {
   adminController.getHotelOwners(req, res, next)
 })
 
-router.put('/owner/accept/:email', async (req, res, next) => {
-  adminController.acceptOwnersEmail(req, res, next)
+router.put('/owner/accept/:id', async (req, res, next) => {
+  adminController.acceptUserToOwner(req, res, next)
 })
 
 router.put('/owner/status/:id', async (req, res, next) => {
-  adminController.verifyOwner(req, res, next)
+  adminController.verifyUser(req, res, next)
```

Comment:
should be owner, we are not veryfying normal users, 
you can add role to this function: 
![image](https://user-images.githubusercontent.com/38506213/110132392-6918d400-7dcb-11eb-890e-79076b1ba368.png)